<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QDialog" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gra</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="headerGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="planningTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>aakar</family>
      <pointsize>26</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="leaveGameBtn">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>30</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Opuść grę</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="infoGroup" stretch="10,75,10,15">
     <item>
      <widget class="QLabel" name="topicLabel">
       <property name="text">
        <string>Temat:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="topicTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="codeShowLabel">
       <property name="text">
        <string>Kod dostępu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="codeLineEdit">
       <property name="maxLength">
        <number>32000</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="playerNameLabel">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>aakar</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gracz</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="footerGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>700</y>
     <width>1600</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>640</y>
     <width>250</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>aakar</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
